lms/envs/common.py

```diff
@@ -1101,6 +1101,9 @@
 # If this is true, random scores will be generated for the purpose of debugging the profile graphs
 GENERATE_PROFILE_SCORES = False
 
+# Sets the number of days after which the gradebook will freeze following the course's end.
+GRADEBOOK_FREEZE_TIMEDELTA = 30
```

Comment:
Could you please change name to `GRADEBOOK_FREEZE_DAYS` and document it following this [guide](https://edx.readthedocs.io/projects/edx-toggles/en/latest/how_to/documenting_new_feature_toggles.html)

Reply:
> document it following this [guide](https://edx.readthedocs.io/projects/edx-toggles/en/latest/how_to/documenting_new_feature_toggles.html)

This is the toggle flag documentation. Shouldn't toggle only be boolean?